src/builders.cjs

```diff
@@ -0,0 +1,44 @@
+const parsers = require('./parsers.cjs');
+
+const RX_SIDE_EFFECTS_IMPORT = parsers.sideEffectsImport();
+const RX_STANDARD_IMPORT = parsers.standardImport();
+const RX_DYNAMIC_IMPORT = parsers.dynamicImport();
+
+const standardImport = ({ dictionary = new Map(), source = '' }) => {
+    const result = source.replace(RX_STANDARD_IMPORT, (replacer, g1, g2, g3, g4) => {
+        const dep = dictionary.get(g4) || g4;
+        return `${g1} ${g2} ${g3} '${dep}'`;
+    });
```

Comment:
I wonder if it would have been better to just have the regexes in this file directly instead of in parsers.cjs. These functions know a bit too much about a regex the developer cannot see directly.

Reply:
Thats a good point. Though we need to export the regexes to test them the way we do so if we put them here or in a separate file does not really matter that much.